src/org/labkey/test/tests/TriggerScriptTest.java

```diff
@@ -448,7 +448,21 @@ public void testDataClassIndividualTriggers() throws Exception
         GoToDataUI goToDataClass = () -> goTo("Data Classes", DATA_CLASSES_NAME);
 
         setupDataClass();
+
+        // Check to see that the logging shows up in the Server JavaScript Console
+        // It's normally accessed via the developer menu, but we can hit the URL directly to avoid dealing with the
+        // popup window in the test
```

Comment:
If this is the only test that touches the sessionLogging page, we should use the menu to ensure that it works correctly.
There was a recent refactor that made the RStudio item in the developer menu not open in a new window (fixed in https://github.com/LabKey/platform/commit/080df3a435d30b5a3a819b3aef4d6add34e7148c).

Reply:
OK. Check out my latest commit. It's passing locally for me. Queued on TeamCity. https://teamcity.labkey.org/buildConfiguration/LabkeyTrunk_DailyCPostgres/3539731